src/decompiler.cc

```diff
@@ -373,21 +385,27 @@ struct Decompiler : ModuleContext {
         }
       }
       case ExprType::Block: {
-        auto le = cast<BlockExpr>(&e);
-        DecompileExprs(le->block.exprs, le->block.decl.GetNumResults(), false);
-        auto &val = stack.back();
-        IndentValue(val, indent_amount, {});
-        val.v.insert(val.v.begin(), "block {");
-        val.v.push_back("}");
+        Block(cast<BlockExpr>(&e)->block, LabelType::Block, "block");
         return;
       }
       case ExprType::Loop: {
-        auto le = cast<LoopExpr>(&e);
-        DecompileExprs(le->block.exprs, le->block.decl.GetNumResults(), false);
-        auto &val = stack.back();
-        IndentValue(val, indent_amount, {});
-        val.v.insert(val.v.begin(), "loop {");
-        val.v.push_back("}");
+        Block(cast<LoopExpr>(&e)->block, LabelType::Loop, "loop");
+        return;
+      }
+      case ExprType::Br: {
+        auto bi = cast<BrExpr>(&e);
```

Comment:
`bi`? Ah, copied from brIf below. Nit; `be` for Br-Expr? (and `bie` for Br-If-Expr)

Also `l` is a questionable name for a variable (looks too much like `1` or `I` at a quick glance), how about `lbl`?

Reply:
Done.